src/data-loader/data-loader.ts

```diff
@@ -280,35 +276,22 @@ const processArticle = (article: ArticleContent): ProcessedArticle => {
 export const loadXmlArticlesFromDirIntoStores = async (dataDir: string, articleRepository: ArticleRepository): Promise<boolean[]> => {
```

Comment:
I think `dataDir` is the unused config:
```
'dataDir' is declared but its value is never read.ts(6133)
```

But following that through is.... sprawling, Sorry.